frame/pallet-account/src/pallet/mod.rs

```diff
@@ -0,0 +1,228 @@
+// This file is part of Astar.
+
+// Copyright (C) 2019-2023 Stake Technologies Pte.Ltd.
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// Astar is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Astar is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Astar. If not, see <http://www.gnu.org/licenses/>.
+
+#[frame_support::pallet]
+#[allow(clippy::module_inception)]
+pub mod pallet {
+    use crate::*;
+
+    use frame_support::pallet_prelude::*;
+    use frame_support::{
+        dispatch::{Dispatchable, GetDispatchInfo},
+        traits::IsSubType,
+    };
+    use frame_system::pallet_prelude::*;
+    use sp_runtime::traits::{IdentifyAccount, Verify};
+
+    /// The current storage version.                                                                                      
+    const STORAGE_VERSION: StorageVersion = StorageVersion::new(1);
+
+    #[pallet::pallet]
+    #[pallet::generate_store(pub(super) trait Store)]
+    #[pallet::storage_version(STORAGE_VERSION)]
+    #[pallet::without_storage_info]
```

Comment:
This should be removed and pallet should correctly be _bounded_ in relations to PoV size.

Reply:
Rewritten to make it bounded.